dom.bs

```diff
@@ -1118,11 +1120,12 @@ and an <a>event listener</a> <var>listener</var>, set <var>listener</var>'s
 method, when invoked, must run these steps:
 
 <ol>
- <li><p>If the <a>context object</a>'s <a>relevant global object</a> is a
- {{ServiceWorkerGlobalScope}} object and its associated <a>service worker</a>'s
- <a for="service worker">script resource</a>'s
- <a for="script resource">has ever been evaluated flag</a> is set, then <a>throw</a> a
- <code>TypeError</code>. [[!SERVICE-WORKERS]]
+ <li><p>If the <a>context object</a> is a {{ServiceWorkerGlobalScope}} object and its associated
+ <a>service worker</a>'s <a for="service worker">script resource</a>'s
+ <a for="script resource">has ever been evaluated flag</a> is set, then
+ <a>report a warning to the console</a> with a description explaining that the user agent will still
+ start the <a>service worker</a> for the <a>service worker event</a> that was synchronously added.
```

Comment:
We don't really know whether it was added at the correct time, right?

Reply:
We are able to know whether the event has been added to the service worker's *set of event types to handle*. But I thought that we'd want to show the warning message when the function is called outside of the first evaluation regardless of whether the event has been added synchronously.

Reply:
The event yes, but not this specific listener.

Reply:
Thinking over it again, I think what we want to achieve for removeEventListener() is to warn, regardless whether it's called sync or async, removing the listener won't effect the service worker start-up if the service worker event was added in the script evaluation. Update the text in the follow-up commit.